### PR TITLE
Megan dashboard

### PR DESCRIPTION
**Pull request by megansam5**
Added template for pages, files for database functions and graphs, as well as working subscribe file

### CODE REVIEW

File: dashboard/Home.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"""Our main home page - that would appear when you click on the link!"""
+import streamlit as st
```
Comment (YakMan101) [inline dashboard/Home.py:2]: Add line after docstring please.

File: dashboard/pages/Topic_Filter.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+"""One page showing a few graphs, where you can filter by topic."""
+import streamlit as st
+
+from db_functions import get_topic_names, get_topic_dict, get_average_score_per_source_for_a_topic, get_title_and_content_data_for_a_topic
+from d_graphs import create_bubble_chart, create_scatter_graph
+
+if __name__ == "__main__":
+
+    st.markdown("<h1 style='color: #e3298c;'>By topic</h1>",
```
Comment (YakMan101) [inline dashboard/pages/Topic_Filter.py:9]: Nice formatting using markdown

File: dashboard/db_functions.py
Comment (indiah444) [inline dashboard/db_functions.py:1]: Really good functions for being able to add and remove subscribers!

File: dashboard/d_graphs.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    """Returns a scatter graph for title vs content score."""
+    scatter_chart = alt.Chart(df).mark_point(filled=True).encode(
+        x=alt.X('title_polarity_score:Q',
+                scale=alt.Scale(domain=[-1, 1]),
+                axis=alt.Axis(title='Title Polarity Score', grid=True)),
+        y=alt.Y('content_polarity_score:Q',
+                scale=alt.Scale(domain=[-1, 1]),
+                axis=alt.Axis(title='Content Polarity Score', grid=True)),
+        color=alt.Color('source_name:N', title='Source'),
+        tooltip=['article_title', 'title_polarity_score',
```
Comment (YakMan101) [inline dashboard/d_graphs.py:31]: you can use tool tip to rename the hints themselves.
e.g:
```
tooltip=[alt.Tooltip(field="article_title", title="Article title"),
              alt.Tooltip(field="title_polarity_score", title="Title polarity score"),
              alt.Tooltip(field="content_polarity_score", title="Content polarity score"),
              alt.Tooltip(field="source_name", title="News Source")]
```

**Review by YakMan101 — COMMENTED** [review]
Good modularity in the way the functions are created

File: dashboard/d_graphs.py (hunk 3 above)
Comment (megansam5) [reply dashboard/d_graphs.py:32]: Ooo thanks I'll do this now!

**Review by indiah444 — APPROVED** [review]
Great improvements made to the dashboard, will be very easy to slot other files into this dashboard to make a really cool final product!

**Pull request merged**